fe/parameterize_simple.py

```diff
@@ -105,27 +119,37 @@ def close(self):
     else:
         raise Exception("precision must be either single or double")
 
-    # host_pdb = app.PDBFile(args.complex_pdb)
-    # modeller = app.Modeller(host_pdb.topology, host_pdb.positions)
-    # modeller.addSolvent(amber_ff, numAdded=6000)
-    # PDBFile.writeFile(modeller.topology, modeller.positions, open("sanitized.pdb", 'w'))
-    # assert 0
-
     suppl = Chem.SDMolSupplier(args.ligand_sdf, removeHs=False)
     for guest_mol in suppl:
         break
 
-    T = 10000
-    dt = 0.0015
-    step_sizes = np.ones(T)*dt
-    cas = np.ones(T)*0.99
+    num_gpus = args.num_gpus
+    num_workers = args.num_gpus*args.jobs_per_gpu
 
-    num_gpus = 1
-    pool = multiprocessing.Pool(num_gpus)
+    print('Creating multiprocessing pool with',args.num_gpus, 'gpus and', args.jobs_per_gpu, 'jobs per gpu')
+    pool = multiprocessing.Pool(num_workers)
 
     all_du_dls = []
 
-    lambda_schedule = np.linspace(0.00001, 0.99999, num=T)
+    start = 1e3
```

Comment:
@chuantian  This is the new lambda schedule

Reply:
It starts by placing the 4th dimension at +1000, then moves it quickly to 1nm in about 500 steps. Then it linearly moves it down to zero. I think we should probably sample the 0.1-0.3 sample more aggressively as well.